samples/Azure.Management.Storage/Generated/Models/AzureEntityResource.cs

```diff
@@ -11,7 +11,7 @@
 namespace Azure.Management.Storage.Models
 {
     /// <summary> The resource model definition for a Azure Resource Manager resource with an etag. </summary>
-    public partial class AzureEntityResource : Resource<ResourceGroupResourceIdentifier>
+    public partial class AzureEntityResource : ResourceManager.Resources.Models.Resource
```

Comment:
We have added Azure.ResourceManager.Resources.Models namespace. Can we remove `ResourceManager.Resources.Models` from here?

Reply:
Its required because we are not omitting the duplicate classes due to ReferenceTypes.  [This item here](https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/6000) will handle it after preview.